pkg/audit/cursor_test.go

```diff
@@ -38,10 +38,17 @@ func TestAuditSegment(t *testing.T) {
 		count int
 	}
 
-	ca, err := testidentity.NewTestCA(ctx)
-	assert.NoError(t, err)
-	identity, err := ca.NewIdentity()
-	assert.NoError(t, err)
+	tctx := testcontext.New(t)
+	defer tctx.Cleanup()
+
+	planet, err := testplanet.New(t, 1, 4, 1)
+	require.NoError(t, err)
+	defer tctx.Check(planet.Shutdown)
+
+	planet.Start(tctx)
+
+	// we wait a second for all the nodes to complete bootstrapping off the satellite
+	time.Sleep(2 * time.Second)
```

Comment:
Timed waits in tests always end up being fragile. Is there not any way to tell when all the nodes have completed bootstrapping?

Reply:
Unfortunately, not yet. Yeah, I don't like it either, but there will be PR-s eventually to fix this.

Reply:
👍